docs/guides/07-zoom-based-styles.md

```diff
@@ -44,23 +43,98 @@ After you explore each one independently, you will bring them together to create
 ### Getting started
 To get started, copy and paste the code below into your favorite text editor and save it to a file `vancouver-trees.html`.
 
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+
+    <script src="../../../dist/carto-vl.js"></script>
+
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.js'></script>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.css' rel='stylesheet' />
+
+    <link rel="stylesheet" type="text/css" href="../../style.css">
+    <style>
+        <!-- Zoom level indicator style -->
+        #js-zoom {
+            position: absolute;
+            bottom: 0;
+            padding: 0 5px;
+            background-color: rgba(255, 255, 255, 0.5);
+            margin: 0;
+            color: rgba(0, 0, 0, 0.75);
+            font-size: 16px;
+        }
+    </style>
+</head>
+
+<body>
+
+    <div id="map"></div>
+
+    <!-- Zoom level indicator container -->
```

Comment:
Same here about the comment and about the blank line

Reply:
I have removed several comments and blank lines here